### PR TITLE
Add note about ad hoc fuzzing and Sweeney reference.

### LINKED ISSUE #217: Add reference for location privacy issues
Comment:

> Section 10: I'd say it'd be good to add a reference to
something that describes some of the privacy issues with
objects such as these, and with potential mitigations, but
more importantly calling out that naively "fuzzing"
boundaries may not be as effective as seems at first the
case. I took a quick look and didn't find anything that
seems really good but maybe something like [1] would be a
good reference.

> [1] http://www.sebastianzimmeck.de/riedererEtAlPhotograph2015ShortPaper.pdf

### PR DESCRIPTION
**Pull request by sgillies**
Resolves #217

### CODE REVIEW

**Comment by sgillies** [comment]
I don't see the perfect reference for discussion of mitigating de-anonymization attacks that exploit location. Sweeney's article is a good introduction to anonymization and well cited, so let's go with it as a starting point.

**Comment by martinthomson** [comment]
I always point people at Section 13.5 of RFC 6772 when they say that they are going to fuzz location.  It is the only good part of the entire document in my opinion: https://tools.ietf.org/html/rfc6772#section-13.5

File: considerations.mkd [hunk 1]
```diff
@@ -4,13 +4,17 @@ GeoJSON shares security issues common to all JSON content types.  See
...
+stored data need protection, which is out of scope for this document.
+
 As with other geographic data formats, e.g., [KMLv2.2], providing
 details about the locations of sensitive persons, animals, habitats, and
-facilities can expose them to unauthorized tracking or injury.  GeoJSON
-does not provide privacy or integrity services.  If sensitive data requires
-privacy or integrity protection, those must be provided by the transport — for
-example TLS or HTTPS. There will be cases in which stored data need protection,
-which is out of scope for this document.
+facilities can expose them to unauthorized tracking or injury. Data
```
Comment (mpdaly) [inline considerations.mkd]: I like the sentiment, but feel that the wording is a bit confusing. How about:

"Data providers should recognize the risk of inadvertantly identifying individuals if anonymized datasets are not adequately skewed or locations are not sufficiently fuzzed [Sweeney]."

**Comment by sgillies** [comment]
@martinthomson @mpdaly thanks! I've pushed a new commit, will merge when the tests pass and the built draft looks good.

**Pull request merged**